src/site/downloads/mac.markdown

```diff
@@ -8,50 +8,43 @@ description: "Installing and updating Dart on your Mac."
 
 # {{ page.title }}
 
-We recommend using Homebrew
+The Dart SDK contains the VM, analyzer, formatter, dart2js compiler,
+documentation generator, package manager, and the core libraries.
+
+We recommend using [Homebrew](#homebrew)
 to manage your Dart installation.
 Another option is [downloading Dart manually](#zip).
 
 
-## Using Homebrew {#homebrew}
+## Installing and upgrading with Homebrew {#homebrew}
 
+Homebrew is a package manager for Mac OS.
 With [Homebrew](http://brew.sh/),
 installing and updating Dart is easy.
-You can use the default location or
-[specify a custom location for Dart](#homebrew-custom-location).
```

Comment:
I was afraid people wouldn't find the custom location section (since it's rather far down) without these words.

But maybe it's just too many words?